lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -634,87 +281,16 @@ def test_calibrated_variance_realizations(self):
 
         cube = self.current_temperature_forecast_cube
 
-        optimised_coeffs = {}
-        the_date = datetime_from_timestamp(cube.coord("time").points)
-        optimised_coeffs[the_date] = np.array(
-            [5, 1, 0, 0.57, 0.6, 0.6], dtype=np.float32)
-
-        self.coeff_names = (
-            ["gamma", "delta", "alpha", "beta0", "beta1", "beta2"])
-
         predictor_cube = cube.copy()
         variance_cube = cube.collapsed("realization", iris.analysis.VARIANCE)
 
-        predictor_of_mean_flag = "realizations"
-
-        plugin = Plugin(cube, optimised_coeffs, self.coeff_names)
-        _, forecast_variance, _ = plugin._apply_params(
-            predictor_cube, variance_cube, optimised_coeffs,
-            self.coeff_names, predictor_of_mean_flag)
+        plugin = Plugin(cube, self.coeffs_from_realizations,
+                        predictor_of_mean_flag="realizations")
+        _, forecast_variance = plugin._apply_params(
+            predictor_cube, variance_cube)
         self.assertArrayAlmostEqual(forecast_variance.data, data,
                                     decimal=4)
 
-    @ManageWarnings(
-        ignored_messages=["Collapsing a non-contiguous coordinate.",
-                          "invalid escape sequence"],
-        warning_types=[UserWarning, DeprecationWarning])
-    def test_coefficients_realizations(self):
```

Comment:
Why has this test been removed?

Reply:
I've removed `test_coefficients` and `test_coefficients_realizations` as I am no longer outputting the coefficients from the `ApplyCoefficientsFromEnsembleCalibration`, as these are now output from `EstimateCoefficientsForEnsembleCalibration`.